### DISCUSSION TITLE
Configuring SkyWalking to Store Data in a Secure Elasticsearch Database

### DISCUSSION THREAD

**Discussion by abhishekacharya828** [apache/skywalking#12122] [General]
In the documentation, it is specified that only insecure Elasticsearch can be used to store data. However, I want to store data in a secure Elasticsearch database. When I change the protocol from HTTP to HTTPS, it throws a configuration error. How can I store data with a secure Elasticsearch database ?

![Screenshot from 2024-04-17 12-36-28](https://github.com/apache/skywalking/assets/134144944/bf9d59a7-9ebf-4b35-8303-81e092428938)

**Comment by wu-sheng** [comment]
Duplicated #12123
